### ISSUE TITLE
Group Props on the Layout

### ISSUE THREAD

**Issue by blargh2947** [smeighan/xLights#3703]
**Is your feature request related to a problem? Please describe.**
I would like to group a prop or set of props on the layout, so that I can move then with the mouse cursor.  
When using props like the EFL Medium NSR tree, the star model is it's own model separate from the tree.  If I move or bump one of them when arranging the layout I need to go back and fix the other.

Also if I have a group of similar props I want to move and preserve the arrangement on the screen I would like them to move as though they were one object.

**Describe the solution you'd like**
After selecting more than one prop, there should be a context menu option for "group" and "ungroup"
If I select group then the selected models should move as one logical model.
If I select ungroup, any previously grouped models should revert to their separate entities. 

**Describe alternatives you've considered**
You can multi select and bump props with the arrow keys, but this can be slow and imprecise.

**Additional context**

**Comment by Strawberry1997** [comment]
Select the group of props, while holding the CTRL key, click and drag.

**Comment by AzGilrock** [comment]
Are you working in 3D or 2D?  I use 3D and its really easy.  Create a group for the models.  When you click on the group name on the left it will select/highlight all the models in the group.  To activate the axis handles hold down ctrl and click one of the models.  Then you can let go of the control key and move the whole group as one using the axis arrowheads.  It would be nice if we auto-selected one of the models in the group to have the focus so you wouldn't need to do the ctrl selection step.

**Comment by computergeek1507** [comment]
I assume he is requesting the ability to "lock" models together for moving not sequence groups.

**Comment by AzGilrock** [comment]
Yep I know you can make a group for either purpose

**Comment by blargh2947** [comment]
> I assume he is requesting the ability to "lock" models together for moving not sequence groups.

Yes correct, to lock a set of props together so they always move as "one"

**Comment by cybercop23** [comment]
similar as #3825

**Comment by derwin12** [comment]
This is in the next update.
